templates/data/mongodb/managed.html

```diff
@@ -340,13 +332,18 @@ <h2>Why choose Managed MongoDB?</h2>
         <h2>Managed MongoDB pricing</h2>
       </div>
       <div class="col-6 col-medium-3">
-        <p>A low total cost of ownership. Up to five times lower than cloud services when fully costed.</p>
+        <p>
+          A low total cost of ownership.
+          <br />
+          Up to five times lower than cloud services when fully costed.
+        </p>
       </div>
     </div>
     <div class="row--50-50">
       <div class="col">
         <hr class="p-rule--highlight" />
-        <div class="p-card--overlay u-no-padding--top" style="min-height: 500px;">
+        <div class="p-card--overlay u-no-padding--top u-no-margin--bottom"
+             style="min-height: 600px">
```

Comment:
Can we remove the custom styling for the card? 
<img width="1268" alt="Screenshot 2024-08-22 at 5 30 15 PM" src="https://github.com/user-attachments/assets/05816748-f9c5-4664-91c1-1f4688c6b938">

Reply:
This has been kept to keep them the same size for different screens.